x-pack/test/apm_api_integration/trial/tests/service_maps/service_maps.ts

```diff
@@ -285,37 +285,5 @@ export default function serviceMapsApiTests({ getService }: FtrProviderContext)
         });
       });
     });
-
-    describe('when there is data with anomalies', () => {
```

Comment:
Why is this test removed?

Reply:
I want to make my backport PR to pass, and it doesn't have any relation to the issue I was solving there. I will open a new PR adding it back.

Reply:
Can you clarify? Not sure why we need to remove this. Do you think the new archive breaks on ES 7.9.0?

Reply:
I added this test on my PR that fixed the error rate, https://github.com/elastic/kibana/pull/75528/files#diff-efea71f9ccdbac05c7ca5d5ac4a993f3R289, I did to make sure that the new `ML_8.0.0` archiver had anomalies. But now that I am going to delete this archive, I'm going to delete also the test, so I can backport the error rate fix to `7.x`.

Reply:
But, this is a test that checks the service maps for anomaly data. We need it for 7.9 as well right? FWIW, I don't expect the backport from this PR to cause issues, because `.ml-state` should no longer be part of the archive. You should be able to use `apm_8.0.0` instead of `ml_8.0.0`. Am I missing something?

Reply:
> this is a test that checks the service maps for anomaly data.
Yeah that's correct, but anyway I'll add the test back, change the archive, and fix it.